doc/whats_new/v1.4.rst

```diff
@@ -1060,12 +1067,12 @@ Larralde, Mateusz Sokół, mathurinm, mecopur, Meekail Zain, Michael Higgins,
 Miki Watanabe, Milton Gomez, MN193, Mohammed Hamdy, Mohit Joshi, mrastgoo,
 Naman Dhingra, Naoise Holohan, Narendra Singh dangi, Noa Malem-Shinitski,
 Nolan, Nurseit Kamchyev, Oleksii Kachaiev, Olivier Grisel, Omar Salman, partev,
-Peter Hull, Peter Steinbach, Pierre de Fréminville, Pooja Subramaniam, Puneeth
-K, qmarcou, Quentin Barthélemy, Rahil Parikh, Rahul Mahajan, Raj Pulapakura,
-Raphael, Ricardo Peres, Riccardo Cappuzzo, Roman Lutz, Salim Dohri, Samuel O.
-Ronsin, Sandip Dutta, Sayed Qaiser Ali, scaja, scikit-learn-bot, Sebastian
-Berg, Shreesha Kumar Bhat, Shubhal Gupta, Søren Fuglede Jørgensen, Stefanie
-Senger, Tamara, Tanjina Afroj, THARAK HEGDE, thebabush, Thomas J. Fan, Thomas
-Roehr, Tialo, Tim Head, tongyu, Venkatachalam N, Vijeth Moudgalya, Vincent M,
-Vivek Reddy P, Vladimir Fokow, Xiao Yuan, Xuefeng Xu, Yang Tao, Yao Xiao,
-Yuchen Zhou, Yuusuke Hiramatsu
+patrickkwang, Peter Hull, Peter Steinbach, Pierre de Fréminville, Pooja
+Subramaniam, Puneeth K, qmarcou, Quentin Barthélemy, Rahil Parikh, Rahul
+Mahajan, Raj Pulapakura, Raphael, Ricardo Peres, Riccardo Cappuzzo, Roman Lutz,
+Salim Dohri, Samuel O. Ronsin, Sandip Dutta, Sayed Qaiser Ali, scaja,
+scikit-learn-bot, Sebastian Berg, Shreesha Kumar Bhat, Shubhal Gupta, Søren
+Fuglede Jørgensen, Stefanie Senger, Tamara, Tanjina Afroj, THARAK HEGDE,
+thebabush, Thomas J. Fan, Thomas Roehr, Tialo, Tim Head, tongyu, Venkatachalam
+N, Vijeth Moudgalya, Vincent M, Vivek Reddy P, Vladimir Fokow, Xiao Yuan,
+Xuefeng Xu, Yang Tao, Yao Xiao, Yuchen Zhou, Yuusuke Hiramatsu
```

Comment:
Please revert, We update the list only once at the time of the release.

Reply:
👍